### PR TITLE
cypress: added tests for comment switching in desktop impress

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: I0cd13c2555274e243ac60cceec0b3e00d2d929ef

* Resolves: #416
* Target version: master 

### Summary
test for comment switching in desktop impress

### CODE REVIEW

File: cypress_test/integration_tests/desktop/impress/comment_switching_spec.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		addSlide(1);
+		addComment(2);
+		changeSlide(1,'previous');
+		cy.get('.leaflet-control-scroll-down').should('exist');
+		cy.get('.leaflet-control-scroll-down').click();
+		cy.get('#PageStatus').should('contain','Slide 2 of 2');
+	});
+
+
+	it('slide should not change when there are multiple comments on same slide', function() {
```
Comment (lpranam) [inline cypress_test/integration_tests/desktop/impress/comment_switching_spec.js:44]: I doubt if this test works correctly! I'll test it.

**Review by lpranam — CHANGES_REQUESTED** [review]
there is no test to check `up` button please add them too. and use smaller test names :)

File: cypress_test/integration_tests/desktop/impress/comment_switching_spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		addSlide(1);
+		addComment(1);
+		changeSlide(1,'previous');
+		addComment(2);
+		var scrollDownButton = cy.get('.leaflet-control-scroll-down').should('exist');
+		for (var n=0;n<2;n++) {
+			cy.get('#PageStatus').should('contain','Slide 1 of 2');
+			scrollDownButton.click();
```
Comment (lpranam) [inline cypress_test/integration_tests/desktop/impress/comment_switching_spec.js:52]: this part is a little fishy, does not matter how many times I iterate test never fails.

File: cypress_test/integration_tests/desktop/impress/comment_switching_spec.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		}
+	});
+});
```
Comment (lpranam) [inline cypress_test/integration_tests/desktop/impress/comment_switching_spec.js:55]: good to have a new line at the end of file

File: cypress_test/integration_tests/desktop/impress/comment_switching_spec.js (hunk 1 above)
Comment (Rash419) [reply cypress_test/integration_tests/desktop/impress/comment_switching_spec.js:44]: Is it working properly?

Comment (lpranam) [reply cypress_test/integration_tests/desktop/impress/comment_switching_spec.js]: nop

**Review by lpranam — APPROVED** [review]

**Comment by lpranam** [comment]
just realised you don't need to add a new impress file you can reuse  `slide_operations.odp`

**Comment by Rash419** [comment]
> just realised you don't need to add a new impress file you can reuse `slide_operations.odp`

Done :)

**Pull request merged**